ibm/mas_devops/roles/suite_config/tasks/main.yml

```diff
@@ -1,17 +1,41 @@
 ---
 
-# 1. Check for undefined properties that do not have a default
+# 1. Check for undefined properties that do not have a default values
 # -----------------------------------------------------------------------------
 - name: "Fail if mas_instance_id is not provided"
   when: mas_instance_id is not defined or mas_instance_id == ""
   fail:
     msg: "mas_instance_id property is required"
 
+- name: "Fail if mas_workspace_id is not provided"
+  when: mas_workspace_id is not defined or mas_workspace_id == ""
+  fail:
+    msg: "mas_workspace_id property is required"
+
 - name: "Fail if mas_config_dir is not provided"
   when: mas_config_dir is not defined or mas_config_dir == ""
   fail:
     msg: "mas_config_dir property is required"
 
+# Wait for Mongocfg CRD
```

Comment:
We can't assume the mongocfg is being applied here.  We do need something like this, but it needs to be more generic and able to handle the different xCfg resources that might (or might not) be passed into this role

Reply:
@santoshjpawar - Can you please on this comment.